packages/react-components/src/components/search-bar/index.tsx

```diff
@@ -1,11 +1,12 @@
-import { css } from '@emotion/core'
+import { css } from '@emotion/react'
+import styled from '@emotion/styled'
 import { giphyBlack, giphyCharcoal, giphyIndigo, giphyLightGrey, giphyWhite } from '@giphy/js-brand'
 import React, { useContext, useEffect, useRef, useState } from 'react'
 import useDebounce from 'react-use/lib/useDebounce'
 import { SearchContext, _SearchContext } from './context'
 import SearchBarChannel from './search-bar-channel'
 import SearchButton from './search-button'
-import styled, { getSize } from './theme'
+import { getSize, PropsWithSearchTheme } from './theme'
```

Comment:
@pshoniuk would it be possible to define the theme this way instead of the `PropsWithSearchTheme` type? 

https://emotion.sh/docs/typescript#define-a-theme

Reply:
I don't think it would work that way, in the package.json we define `"types": "dist/index.d.ts"` and I think we'd have to export the emotion.d.ts from there to change the types in the host project. 

Tested with the following `src/emotion.d.ts` in the react package 

```tsc
import '@emotion/react'
import { SearchTheme } from './components/search-bar/theme'

declare module '@emotion/react' {
    export interface Theme extends SearchTheme {}
}
```

and tsconfig `"files": ["src/index.tsx", "src/emotion.d.ts"]` 

and it works pretty well in the package 

but that said, it's not super important so up to you

Reply:
Cool, thanks for the explanation. I updated the PR